website/docs/r/grafana_workspace_api_key.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+subcategory: "Managed Grafana"
+layout: "aws"
+page_title: "AWS: aws_grafana_workspace_api_key"
+description: |-
+  Creates a Grafana API key for the workspace. This key can be used to authenticate requests sent to the workspace's HTTP API.
+---
+
+# Resource: aws_grafana_workspace_api_key
+
+Provides an Amazon Managed Grafana workspace API Key resource.
+
+## Example Usage
+
+### Basic configuration
+
+```terraform
+resource "aws_grafana_workspace_api_key" "key" {
+  key_name        = "test-key"
+  key_role        = "VIEWER"
+  seconds_to_live = 3600
+  workspace_id    = aws_grafana_workspace.test.id
+}
+```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `editor_role_values` - (Required) The editor role values.
```

Comment:
The valid values for Role can be defined here. From the documentation at https://docs.aws.amazon.com/grafana/latest/APIReference/API_CreateWorkspaceApiKey.html, the valid values are `VIEWER`|`EDITOR`|`ADMIN`